packages/graphiql/src/components/QueryEditor.js

```diff
@@ -69,6 +75,7 @@ export class QueryEditor extends React.Component {
     require('codemirror/addon/dialog/dialog');
     require('codemirror/addon/lint/lint');
     require('codemirror/keymap/sublime');
+    require('codemirror/keymap/vim');
```

Comment:
possibly do inline requires based on the chosen option? These files can be pretty large. pretty soon we will also replace all these requires with `import().then()`

Reply:
For the toggling between vim/sublime modes, we need to have these files to be preloaded.

Reply:
@semanser i think we're ready to accept this earlier than we though, but we really need to use `import()`, as we are switching all of this to use `import()` very soon. it will be pre-loaded for the minified bundle, and loaded async for more modern setups. it will work great i promise. its a great use case for contextually loading modules asynchronously, with sublime as the one thats loaded by default by the entrypoint bundle. then the VIM module makes no impact on the entrypoint bundle size, though it still impacts the min